config/locales/en.yml

```diff
@@ -2356,3 +2354,16 @@ en:
           - "(if known) 'We estimate [X] checks were affected by this issue'"
         content: Resolution text
         resolve: Resolve problem
+  feedback:
+   freetext:
```

Comment:
This indentation looks fractionally off - I think you need to indent this line and the stuff below it by one more space.